README.md

```diff
@@ -34,3 +34,38 @@ dataset = app.create_dataset(dataset_id="demo_dataset")
 # execute data upload to Clarifai app dataset
 dataset.upload_dataset(task='visual_segmentation', split="train", dataset_loader='coco_segmentation')
 ```
+
+## Interacting with Models
+
+### Model Predict
+```python
+from clarifai.client.model import Model
+
+# Model Predict
+model = Model(user_id="user_id", app_id="app_id", model_id="model_id")
+model_prediction = model.predict_by_url(url="url", input_type="image") # Supports image, text, audio, video
+
+# Customizing Model Inference Output
+model = Model(user_id="user_id", app_id="app_id", model_id="model_id",
+                  output_config={"min_value": 0.98}) # Return predictions having prediction confidence > 0.98
+model_prediction = model.predict_by_filepath(filepath="local_filepath", input_type="text") # Supports image, text, audio, video
+
+model = Model(user_id="user_id", app_id="app_id", model_id="model_id",
+                    output_config={"sample_ms": 2000}) # Return predictions for specified interval
+model_prediction = model.predict_by_url(url=BEER_VIDEO_URL, input_type="video")
+```
+### Models Listing
+```python
+# List all model versions
+all_model_versions = model.list_versions()
+
+# Go to specific model version
+model_v1 = client.app("app_id").model("model_id").version("model_version_id")
```

Comment:
when we do this kind of chained request, is it making an API call for each function call here?

because the alternative would be Model(user_id, app_id, model_id, version_id) which would of course only use one API call. I wonder if we need to add caching in the client with a TTL so that both end up being as efficient?

Reply:
Yes, for chained request it makes API call for each function(uses `GetApp/Model/DatasetRequest` endpoint). Will discuss with team on adding TTL going further.